src/Microsoft.Diagnostics.Monitoring.Options/GlobalCounterOptionsDefaults.cs

```diff
@@ -10,6 +10,6 @@ namespace Microsoft.Diagnostics.Monitoring.WebApi
 {
     internal static class GlobalCounterOptionsDefaults
     {
-        public const int IntervalSeconds = 5;
+        public const float IntervalSeconds = 5;
```

Comment:
```suggestion
        public const float IntervalSeconds = 5.0f;
```
Just make it obvious by reading that this can be a float.